Rakefile

```diff
@@ -1,8 +1,13 @@
 # frozen_string_literal: true
 require 'rubocop/rake_task'
+require 'rake/testtask'
 RuboCop::RakeTask.new
 
 require 'scraper_test'
 ScraperTest::RakeTask.new.install_tasks
 
+Rake::TestTask.new do |t|
```

Comment:
I've added a commit to update the scraper_test gem. To make use of https://github.com/everypolitician/scraper_test/pull/8